### ISSUE TITLE
Definition of conformance metadata in principles document is not abstract

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#16]
The definition of accessibility conformance metadata is written from perspective of dc:conformsto, which uses URL, on the other hand ONIX uses codes. This definition should be rewritten in abstract way. Copying the text below:
To report the accessibility conformance of a publication, the metadata term conformsTo must be provided. The conformsTo term requires a specific URL that identifies the level of conformance of the publication. The values for an EPUB publication are:
* http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a
* http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa
* http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-
The above URLs indicate that the publication meets the EPUB accessibility requirements and the corresponding WCAG levels of A, AA, and AAA.

If a publication is optimized for a particular user group, e.g. an audiobook, it would not conform to WCAG guidelines, but it may be perfectly useable by a particular group, e.g. persons who are blind. In this case, the conformsTo term must point to a URL that identifies the specification used to create the optimized publication.

The improved version is as follows. Further improvements are welcome.
To report the accessibility conformance of a publication, metadata for accessibility conformance should be provided. It declares the accessibility standard or accessibility specifications to which the publication conforms. The value can be a URL or a code which identifies the accessibility standard or accessibility specifications.

In case of EPUB publications, it will usually point to the EPUB Accessibility specifications or WCAG. If a publication is optimized for a particular user group, e.g. an audiobook, it would not conform to WCAG guidelines, but it may be perfectly useable by a particular group, e.g. persons who are blind. In this case, the accessibility conformance metadata should identify the specification used to create the optimized publication.
